### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed

### DISCUSSION THREAD

**Discussion by hankchan** [community/community#66730] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

[INFO] [auth] [2023-09-11T00:56:35.400Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-09-11T00:56:35.406Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as hankchan."
[INFO] [auth] [2023-09-11T02:27:37.865Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-09-11T02:27:37.868Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as hankchan."

**Comment by davecheney** [comment, marked as answer]
There is a problem with your billing, please contact support https://support.github.com/contact who can assist you.
